config/puma/production.rb

```diff
@@ -1,3 +1,8 @@
+run_dir = "/run/foreman"
```

Comment:
I think this currently breaks running a development checkout when using `RAILS_ENV=production`. In our packaged setup we symlink `tmp` to this directory so you can make a compatible change:
```suggestion
run_dir = Rails.root.join('tmp')
```

Reply:
To understand this better, can you explain the benefit of this change?